src/views/movie_list_view.js

```diff
@@ -0,0 +1,56 @@
+import Backbone from 'backbone';
+import Movie from '../models/movie';
+import MovieList from '../collections/movie_list';
+import MovieView from '../views/movie_view';
+import ReturnedMovieView from '../views/returned_movie_view';
+import $ from 'jquery';
+
+const MovieListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.listenTo(this.model, 'update', this.render);
+    this.listenTo(this.bus, 'addMovie', this.addToCollection);
+
+    // this.listenTo(this.bus, 'addToCollection', this.addToCollection);
+  },
+  render() {
+    this.$('#movie-list').empty();
+
+    const lastMovie = this.model.at(this.model.length -1);
+    const movieView = new MovieView({
+      tagName: 'tr',
+      template: this.template,
+      model: lastMovie,
+      bus: this.bus,
+    });
+
+    this.$('#movies-in-store').append(movieView.render().$el);
+    return this;
+  },
+  addToCollection(attributes) {
+    const newMovie = new Movie(attributes);
+    if (newMovie.isValid()) {
+      const movieView = new MovieView({
```

Comment:
Instead of creating a `MovieView` here and appending its HTML directly to the DOM, it would probably make more sense to add the `newMovie` to the movie list. That would trigger an `update` event, which would be handled by `render`, resulting in the same thing.

Aside from being DRYer, the fact that you _don't_ add the movie to the list here means that if this view re-renders for some other reason, the newly added movie will be absent.